tests/data_result/module_130_140/views/sale_order.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+  <record id="act_window_example" model="ir.actions.act_window">
+    <field name="name">Example</field>
+    <field name="res_model">account.move.line</field>
+    <field name="binding_model">account.move</field>
```

Comment:
Shall it work? I'm getting this error:

odoo_1                    | ValueError: Invalid field 'binding_model' on model 'ir.actions.act_window'

Reply:
It should be 

```
            <field name="binding_model_id" ref="account.model_account_move"/>
```


https://github.com/odoo/odoo/blob/3c5296863f0030acc5d3dba0ff73a6f1b42d1255/addons/account/views/account_analytic_default_view.xml#L111-L118

Reply:
That replacement is not due to the modifications in this PR: it comes from https://github.com/OCA/odoo-module-migrator/blob/6a9aa1d39888ef6dbe44e1675db7373e75dee9be/odoo_module_migrate/migration_scripts/migrate_120_130.py#L36
I've created https://github.com/OCA/odoo-module-migrator/issues/52 to keep track of the issue.

Reply:
Actually, scratch that, I was confused by v13's `act_window` tag.
Now I double checked and:

- In v13, `act_window` tag actually has an attribute `binding_model`that was then mapped to `ir.actions.act_window.binding_model_id`, so I've closed the issue. 
- In v14, the field `ir.actions.act_window.binding_model` (like in v13) does not exist and should actually be `ir.actions.act_window.binding_model_id`

So I'll fix this PR so that the v13 -> v14 migration takes care of all this.

Reply:
By the way, something strange was definitely happening because the migration scripts from previous versions were being executed and the 12.0_13.0 scripts was actually responsible of the `binding_model` substitution.

I fixed that in a dedicated commit below (the one about class/instance attributes), could you have a look and update your review?